titanic.ipynb

```diff
@@ -163,13 +159,19 @@
    },
    "outputs": [],
    "source": [
-    "# Wait for the generation of the enclave identity\n",
+    "# Wait for the generation of the enclave identity, we are using the code provider object to do the waiting but \n",
```

Comment:
Could be interesting to precise that they can access stdout and stderr ? Or too much for demo, just in the doc ?

Reply:
@seiteta What do you think of this?